search_engine_parser/core/base.py

```diff
@@ -11,141 +11,146 @@
 
 
 class BaseSearch(object):
```

Comment:
Why does your code editor rewrites the entire file? it kinda makes it difficult to locate changed lines

Reply:
Sure, I figured out the issue. Will be changed back.

Reply:
You should reset the file to the origin commit and force push. You can use `checkout <commit-shasum> -- search_engine_parser/core/base.py` to reset it to commit-shasum.